### PR TITLE
add support for 2 alecto devices

### PR DESCRIPTION
**Pull request by JoshuaDodds**

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -8060,6 +8060,31 @@ const devices = [
         description: 'FLAIR Viyu Smarte LED bulb RGB E27',
         extend: generic.light_onoff_brightness_colortemp_colorxy,
     },
+
+    // Alecto Smart Home
+    {
+        // this device needs its contact return value inverted from what 'fz.ias_contact_alarm_1' to work with domoticz
```
Comment (Koenkk) [inline devices.js]: If the contact is closed (so magnetic parts are against each other), what is the value of `contact`, `TRUE` or `FALSE`?

Comment (JoshuaDodds) [reply devices.js]: This is the debug value of close:

```
debug 2020-02-29 15:44:43: Received Zigbee message from 'Door_Sensor', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"zonestatus":0,"extendedstatus":0}' from endpoint 1 with groupID 0
```

and open then is:

```
debug 2020-02-29 15:44:43: Received Zigbee message from 'Door_Sensor', type 'commandStatusChangeNotification', cluster 'ssIasZone', data '{"zonestatus":1,"extendedstatus":0}' from endpoint 1 with groupID 0
```

Comment (Koenkk) [reply devices.js]: Then the added converter is not correct, you can just use `ias_contact_alarm_1`. When the contact is closed, contact should be TRUE (as contact is made), on open there is no contact, so contact should be FALSE. This is how it's currently implemented for all contact sensors.

**Comment by Koenkk** [comment]
Note to self: after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/229

File: devices.js (hunk 1 above)
Comment (JoshuaDodds) [reply devices.js]: @Koenkk I would be inclined to agree but when i invert the value returned in the converter both the domoticz python zigbee2mqtt plugin and the homebridge siri integration project work as expected with this device.   Do you think the inversion of behavior should actually be done in the python plugin then ?

Comment (Koenkk) [reply devices.js]: It should indeed be done there, it's important to have this value consistent within zigbee2mqtt otherwise it becomes unmanageable

Comment (JoshuaDodds) [reply devices.js]: I agree!  Will fix this with your input and push the changes this weekend.

**Comment by Koenkk** [comment]
Thanks!

**Comment by Koenkk** [comment]
It seems that a device with this zigbeemodel is already supported: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L8115.

Could you provide the `database.db` entry of this device? (`RH3001`)

**Comment by JoshuaDodds** [comment]
> 
> 
> It seems that a device with this zigbeemodel is already supported: https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices.js#L8115.
> 
> Could you provide the `database.db` entry of this device? (`RH3001`)

{"id":28,"type":"EndDevice","ieeeAddr":"0x086bd7fffeed887e","nwkAddr":56770,"manufId":0,"manufName":"TUYATEC-kbqf60nt","powerSource":"Battery","modelId":"RH3001","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":1026,"inClusterList":[0,1,3,1280],"outClusterList":[3],"clusters":{"genBasic":{"attributes":{"modelId":"RH3001","manufacturerName":"TUYATEC-kbqf60nt","powerSource":3,"zclVersion":2,"appVersion":74,"stackVersion":0,"hwVersion":1,"dateCode":"20180608"}},"genPowerCfg":{"attributes":{"batteryVoltage":30,"batteryPercentageRemaining":200}}},"binds":[]}},"appVersion":74,"stackVersion":0,"hwVersion":1,"dateCode":"20180608","zclVersion":2,"interviewCompleted":false,"meta":{},"lastSeen":1583298457575}


and the other one is:

{"id":38,"type":"EndDevice","ieeeAddr":"0x14b457fffe16bff0","nwkAddr":58376,"manufId":0,"manufName":"TUYATEC-abkehqus","powerSource":"Battery","modelId":"RH3052","epList":[1],"endpoints":{"1":{"profId":260,"epId":1,"devId":770,"inClusterList":[0,1,3,1026,1029],"outClusterList":[3,1026,1029],"clusters":{"genBasic":{"attributes":{"modelId":"RH3052","manufacturerName":"TUYATEC-abkehqus","powerSource":3,"appVersion":0,"zclVersion":2,"stackVersion":0,"hwVersion":1,"dateCode":"20180608"}},"genPowerCfg":{"attributes":{"batteryVoltage":30,"batteryPercentageRemaining":200}},"msTemperatureMeasurement":{"attributes":{"measuredValue":1846}},"msRelativeHumidity":{"attributes":{"measuredValue":4563}}},"binds":[]}},"appVersion":0,"stackVersion":0,"hwVersion":1,"dateCode":"20180608","zclVersion":2,"interviewCompleted":true,"meta":{},"lastSeen":1583297850441}

**Comment by Koenkk** [comment]
Both devices seem to be rebranded versions of the TUYA devices,that means this device should already work in zigbee2mqtt 1.11.0 out-of-the-box. can you check this?

**Comment by JoshuaDodds** [comment]
> 
> 
> Both devices seem to be rebranded versions of the TUYA devices,that means this device should already work in zigbee2mqtt 1.11.0 out-of-the-box. can you check this?

No they did not work out of the box.  If they did I would not have added them.  I am running 1.11.0

**Comment by Koenkk** [comment]
With a clean zigbee2mqtt 1.11.0, can you pair the devices and provide the log?

**Pull request merged**